extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -650,7 +649,6 @@ export class AmpAnalytics extends AMP.BaseElement {
       resultPromises.push(
```

Comment:
should we get rid of this array if we are not doing anything with it?

Reply:
good catch. removed.